connexion/json_schema.py

```diff
@@ -13,9 +14,11 @@
 
 import requests
```

Comment:
I think the imports are grouped (1) python standard, (2) third party and (3) connexion.  FWIW maybe it's worth sorting within categories (1) and (2) here?

Reply:
There is a sorting within categories: At first normal `imports` sorted alphabetically, then `from` imports sorted alphabetically.

If I change the order `isort` will complain.

Reply:
Thanks for explaining, I didn't spot that pattern, never mind!